### ISSUE TITLE
Translate service.iptv.manager

### ISSUE THREAD

**Issue by michaelarnauts** [add-ons/translations#10]
We would like to request translations for add-on **IPTV Manager**.

You can find the translations at: https://github.com/add-ons/service.iptv.manager/tree/master/resources/language

Here is the list of Kodi-addons languages and their status.
*(See below for an error log)*

|Legend|Meaning
|--------|--------------------------
|✔️|Translation complete
|⚠️|Missing strings
|❌|No translations

|Language|ISO|Status|Last translator|PR
|-----------|----|-------|-----------------|-------------------------------
|Bosnian|bs_ba|❌|
|Bulgarian|bg_bg|❌|
|Montenegrin|cnr_me|❌|
|Czech|cs_cz|❌|
|Danish|da_dk|❌|
|German|de_de|❌|
|Greek|el_gr|✔️|@Twilight0
|**English**|en_gb| | |*(default language)*
|Spanish|es_es|❌|
|Mexican|es_mx|❌|
|Finnish|fi_fi|❌|
|French|fr_fr|❌|
|Hebrew|he_il|❌|
|Croatian|hr_hr|❌|
|Hungarian|hu_hu|✔️|@takraj
|Hindi|in_hi|❌|
|Italian|it_it|❌|
|Japanese|ja_jp|❌|
|Korean|ko_kr|❌|
|Lithuanian|lt_lt|❌|
|Latvian|lv_lv|❌|
|Macedonian|mk_mk|❌|
|Norwegian|nb_no|❌|
|Dutch|nl_nl|✔️|@michaelarnauts
|Polish|pl_pl|❌|
|Brazilian|pt_br|❌|
|Portuguese|pt_pt|❌|
|Romanian|ro_ro|✔️|@tmihai20
|Russian|ru_ru|✔️|@vlmaksime
|Serbian|sr_rs|❌|
|Slovak|sk_sk|❌|
|Slovenian|sl_si|❌|
|Swedish|sv_se|❌|
|Turkish|tr_tr|❌|
|Taiwanese|zh_tw|❌|

> NOTE: Not all past translators have accepted the invitation yet.

<details>
  <summary>Translation errors</summary>
  <br/><p><pre>

```
```
</pre></p>
</details>

cc @add-ons/translators

**Comment by Twilight0** [comment]
Added greek translation: https://github.com/add-ons/service.iptv.manager/pull/62

**Comment by dagwieers** [comment]
Apart from translations, I think there is a convincing story to ask other Kodi add-ons to implement this.
Maybe we can start listing them at: https://github.com/add-ons/service.iptv.manager/issues/63

If you are using Kodi add-ons in your region that are tied to broadcasted/live channels, feel free to list them or discuss IPTV Manager integration with the author. It is quite easy to integrate Live channels in Kodi's TV/Radio section, and if you have EPG data integrate the VOD content in Kodi's TV Guide.

**Comment by michaelarnauts** [comment]
If somebody wants to translate https://github.com/add-ons/service.iptv.manager/wiki/Installation-(NL) to English, or your own language, please go ahead. You can just attach it here to this issue, and I'll make sure it ends up on the wiki.

Thanks!

**Comment by Twilight0** [comment]
> Apart from translations, I think there is a convincing story to ask other Kodi add-ons to implement this.
> Maybe we can start listing them at: [add-ons/service.iptv.manager#63](https://github.com/add-ons/service.iptv.manager/issues/63)
> 
> If you are using Kodi add-ons in your region that are tied to broadcasted/live channels, feel free to list them or discuss IPTV Manager integration with the author. It is quite easy to integrate Live channels in Kodi's TV/Radio section, and if you have EPG data integrate the VOD content in Kodi's TV Guide.

Indeed, I'd love to test in my own case on my flagship addon (AliveGR), which already had IPTV integration, but I temporarily disabled it due to instability.

**Comment by tmihai20** [comment]
Romanian translation awaits with PR https://github.com/add-ons/service.iptv.manager/pull/65.
